hummingbot/connector/exchange/coinbase_advanced_trade/coinbase_advanced_trade_exchange.py

```diff
@@ -0,0 +1,1107 @@
+import asyncio
+import decimal
+import logging
+import math
+from decimal import Decimal
+from typing import TYPE_CHECKING, Any, AsyncGenerator, AsyncIterable, Dict, Iterable, List, Tuple
+
+from async_timeout import timeout
+from bidict import bidict
+
+from hummingbot.connector.constants import s_decimal_NaN
+from hummingbot.connector.exchange_py_base import ExchangePyBase
+from hummingbot.connector.time_synchronizer import TimeSynchronizer
+from hummingbot.connector.trading_rule import TradingRule
+from hummingbot.connector.utils import TradeFillOrderDetails
+from hummingbot.core.data_type.cancellation_result import CancellationResult
+from hummingbot.core.data_type.common import OrderType, TradeType
+from hummingbot.core.data_type.in_flight_order import InFlightOrder, OrderState, OrderUpdate, TradeUpdate
+from hummingbot.core.data_type.order_book_message import OrderBookMessage
+from hummingbot.core.data_type.order_book_tracker_data_source import OrderBookTrackerDataSource
+from hummingbot.core.data_type.trade_fee import AddedToCostTradeFee, TokenAmount, TradeFeeBase
+from hummingbot.core.data_type.user_stream_tracker_data_source import UserStreamTrackerDataSource
+from hummingbot.core.event.events import MarketEvent, OrderFilledEvent
+from hummingbot.core.utils.async_utils import safe_gather
+from hummingbot.core.utils.estimate_fee import build_trade_fee
+from hummingbot.core.web_assistant.web_assistants_factory import WebAssistantsFactory
+from hummingbot.logger import HummingbotLogger
+
+from . import coinbase_advanced_trade_constants as constants, coinbase_advanced_trade_web_utils as web_utils
```

Comment:
use abs imports as std in the codebase

Reply:
Rational for relative imports:
   - helps separate external from internal to the module. Would it be more helpful to put them in the __init__.py?
   - cosmetic: coinbase_advanced_trade is a long name that makes things unreadable, in particular when they get mixed-up with other `hummingbot....` long names
   - I did check and found several other modules using this improvement
   
Apart from 'std in the codebase' are there other drawbacks, I did some research on relative imports and did not find a strong argument that would make hummingbot code less robust - This is not to mean 'std in the codebase' is not suffucient, but rather for my own education on learning python

Reply:
Updated